js/about/preferences.js

```diff
@@ -686,6 +686,21 @@ class PaymentsTab extends ImmutableComponent {
     </div>
   }
 
+  get nextReconcileDate () {
+    // Show date only after the user has funded the wallet.
+    // Afterwards balance should always !== 0 due to exchange rate fluctuations.
+    let ledgerData = this.props.ledgerData
+    let balance = Number(ledgerData.get('balance') || 0)
+    if (balance === 0 || !ledgerData.get('reconcileStamp')) {
+      return null
+    }
+    let nextReconcileDate = formattedDateFromTimestamp(ledgerData.get('reconcileStamp'))
+    let l10nDataArgs = {
+      reconcileDate: nextReconcileDate
+    }
+    return <div className='nextReconcileDate' data-l10n-args={JSON.stringify(l10nDataArgs)} data-l10n-id='statusNextReconcileDate' />
```

Comment:
we generally prefer `const` over `let` for declaring variables that aren't going to be reassigned